todoman/formatters.py

```diff
@@ -82,6 +82,8 @@ def detailed(self, todo):
             extra_rows += self._columnize('Description', todo.description)
         if todo.location:
             extra_rows += self._columnize('Location', todo.location)
+        if todo.priority:
```

Comment:
This is not necessary (and the corresponding test as well), because the priority is already shown as part of `self.compact(todo)`